ci/jobs/build_clickhouse.py

```diff
@@ -104,6 +100,13 @@ def main():
 
     cmake_cmd = BUILD_TYPE_TO_CMAKE[build_type]
     info = Info()
+    if not info.is_local_run:
+        os.environ["SCCACHE_BUCKET"] = Settings.S3_ARTIFACT_PATH
+        os.environ["SCCACHE_S3_KEY_PREFIX"] = "ccache/sccache"
+        os.environ["CTCACHE_DIR"] = "ccache/clang-tidy-cache"
+        os.environ["CTCACHE_S3_BUCKET"] = Settings.S3_ARTIFACT_PATH
+        os.environ["CTCACHE_S3_FOLDER"] = "clang-tidy-cache"
```

Comment:
Likely it should be the same `ccache/clang-tidy-cache` (since I see only this path in S3) - https://github.com/ClickHouse/ClickHouse/pull/78471